scripts/firmware.py

```diff
@@ -640,18 +695,74 @@ def set_custom_branch(self):
             logging.info("No custom branch provided.")
             self.branch_menu()
 
-    def restart_klipper(self):
+    def edit_config(self, config: str) -> None:
```

Comment:
```suggestion
    def edit_config(self, option: str) -> None:
```
config in config sounds weird 😂